versiondb/client/dump.go

```diff
@@ -201,7 +201,10 @@
 		return err
 	}
 
-	return writer.Flush()
+	if err := writer.Flush(); err != nil {
+		return err
+	}
+	return fp.Sync()
```

Comment:
But call sync on these temporary files hurt performance for no good though.

Reply:
so we can only detect invalid file when read

Reply:
if the process don't finish succesfully, user just need to rerun.